src/backend/src/routes/reimbursement-requests.routes.ts

```diff
@@ -24,6 +24,8 @@ reimbursementRequestsRouter.get('/reimbursements/current-user', ReimbursementReq
 
 reimbursementRequestsRouter.get('/reimbursements', ReimbursementRequestController.getAllReimbursements);
 
+reimbursementRequestsRouter.post('/edit-vendors', nonEmptyString(body('name')), ReimbursementRequestController.editVendor);
```

Comment:
Also, let's update the endpoint path to `/:vendorId/vendors/edit` so we can get vendorId from `req.params` in the edit vendor controller function.